### PR TITLE
Convert all projects to scoped registries and upgrade BasicSample to 2021

### PR DESCRIPTION
**Pull request by KBonds**

### CODE REVIEW

File: BasicSample/Packages/manifest.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-    "com.microsoft.mixedreality.toolkit.standardassets": "file:../../MixedRealityPackages/com.microsoft.mixedreality.toolkit.standardassets-2.8.3.tgz",
+    "com.microsoft.mixedreality.toolkit.foundation": "2.8.3",
+    "com.microsoft.mixedreality.toolkit.standardassets": "2.8.3",
     "com.microsoft.mrtk.graphicstools.unity": "https://github.com/microsoft/MixedReality-GraphicsTools-Unity.git?path=/com.microsoft.mrtk.graphicstools.unity#v0.5.0",
-    "com.unity.ide.rider": "3.0.17",
+    "com.unity.ide.rider": "3.0.18",
     "com.unity.ide.visualstudio": "2.0.17",
     "com.unity.ide.vscode": "1.2.5",
-    "com.unity.inputsystem": "1.4.4",
+    "com.unity.inputsystem": "1.5.0",
```
Comment (KBonds) [inline BasicSample/Packages/manifest.json:11]: Revert to 1.4.4

Comment (KBonds) [reply BasicSample/Packages/manifest.json]: done!

**Comment by KBonds** [comment]
This last commit made the diff too large to realistically read - if you're taking a look at this, filtering to the first two commits is easier to see the changes: https://github.com/microsoft/OpenXR-Unity-MixedReality-Samples/pull/145/files/c2219f194b1448a7b8fc1191d4b193764f0ec1bc

Other than this, the project folder was just duplicated and the project names were changed.

**Review by devdeeprc — APPROVED** [review]

**Comment by devdeeprc** [comment]
In one of the commits, seems input system was downgraded. Thats no longer needed. You can upgrade to 1.5.1

**Review by devdeeprc — APPROVED** [review]

**Pull request merged**